File: swift/iniciante/1017.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(),
   let tempo = Double(inputA.replacingOccurrences(of: ",", with: ".")),
   let velocidade = Double(inputB.replacingOccurrences(of: ",", with: "."))
{
    let distancia = tempo * velocidade
    let litros = distancia / 12

    print(String(format: "%.3f", litros))
}

File: typescript/iniciante/1017.ts
import { bold, cyan } from "https://deno.land/std@0.224.0/fmt/colors.ts";

const tempo = Number(prompt("Digite o tempo (horas):"));
const velocidade = Number(prompt("Digite a velocidade média (km/h):"));

const distancia = tempo * velocidade;
const litros = distancia / 12;

console.log(cyan(bold(litros.toFixed(3))));
